roles/preflight/README.md

```diff
@@ -57,96 +59,178 @@ The idea is use one of them.
+        organization_id: "12345678"
+
+        # Reduce the job duration
+        do_must_gather: false
+        check_workload_api: false
+        preflight_run_health_check: false
+
+        # Credentials for your private registries.
+        # You could have several private registries:
+        # local and another external, to store the operator.
+        # In this case, please provide all credentials here.
+        partner_creds: "/opt/pull-secrets/partner_config.json"
+
+        # Optional; provide it when you need to submit test results.
+        # This token is shared between all your projects.
+        # To generate it: connect.redhat.com -> Product certification ->
+        # Container API Keys -> Generate new key
+        pyxis_apikey_path: "/opt/cache/pyxis-apikey.txt"
+
+        # Mandatory for disconnected environment,
+        # this registry is used for mirrored images
+        # and to store an index (catalog) image.
+        dci_local_registry: registry.local.lab:4443
+
+        # Optional, provide it if your registry is self-signed.
+        preflight_custom_ca: "/var/lib/dci-openshift-agent/registry/certs/cert.ca"
+
+        # List of images to certify,
+        # you could provide many containers at once.
+        preflight_containers_to_certify:
+            # In disconnected environments provide a digest (SHA), not a tag.
+          - container_image: "quay.io/rh-nfv-int/testpmd-operator@sha256:339096a68f09eb42aa9bb8d72bf1d958fe14a15c93fb850e27ce18208fed23ed"
+            # Required when creating cert project but optional if not want to update mandatory parameters
+            short_description: "Add 50+ characters image description here"
+            # Optional; provide it when you need to create
+            # a new "Container Image project" and submit test results in it.
+            create_container_project: true
+
+        # Project certification setting (Optional)
+        # This allows to fill the rest of the project settings after project creation
+        # Any project for containers images certifications can use them
+        # TODO: provide cert_settings example for every project type
+        cert_settings:
+          # Operator and Container
+          auto_publish: false
+          registry_override_instruct: "These are instructions of how to override settings"
+          email_address: "email@email.com"
+          application_categories: "Networking"
+          privileged: false
+          repository_description: "This is a test repository"
+          build_categories: "Standalone image"
+          os_content_type: "Red Hat Universal Base Image (UBI)"
+          release_category: "Generally Available"
+
+      use_previous_topic: true
+      inputs:
+        kubeconfig: kubeconfig_path
+    ...
     ```
 
 - Run dci-openshift-app-agent:
 
     ```bash
-    dci-openshift-app-agent-ctl -s -- -v
+    KUBECONFIG=$KUBECONFIG dci-pipeline-schedule testpmd-container-disconnected
```

Comment:
This is just to run the second case (disconnected), but not valid for the first case